app/src/main/java/me/guillaumin/android/osmtracker/db/model/TrackStatistics.java

```diff
@@ -0,0 +1,246 @@
+package me.guillaumin.android.osmtracker.db.model;
+
+import me.guillaumin.android.osmtracker.db.TrackContentProvider;
+import me.guillaumin.android.osmtracker.db.TrackContentProvider.Schema;
+import android.content.ContentResolver;
+import android.database.Cursor;
+import android.location.Location;
+import android.util.Log;
+
+/**
+ * Represents statistics, such as  total length and maximum speed, for a Track
+ *
+ * @author Arseniy Lartsev
+ *
+ */
+public class TrackStatistics {
+	private long trackId;
+	private long pointCount;
+	private float length;
+	private float maxSpeed;
+	private double lastLatitude;
+	private double lastLongitude;
+	private long lastTime;
+	private float timeMoving;
+
+	/**
+	 * Copypasta from osmand
```

Comment:
Searching for 'osmand' on maveng.org yields nothing, couldn't find anything relevant while searching for 'library' on osmand.net. I guess they don't?